.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,27 @@
+name: E2E Tests
+
+on:
+    pull_request:
+    push:
+        branches:
+            - trunk
+jobs:
+    e2e:
+      runs-on: ubuntu-latest
+      strategy:
+            matrix:
+                node-version: [14.x, 16.x]
```

Node 14 is going to become legacy at the end of April, Node.js 18 is now the current line:
```suggestion
                node-version: [16.x, 18.x]
```